sites/statics/server.js

```diff
@@ -0,0 +1,18 @@
+var path = require("path");
+var express = require("express");
+var fallback = require("express-history-api-fallback");
+
+// Create application
+var app = express();
+
+// Set public directory
+var publicDir = path.resolve(__dirname);
+
+// Set static application options
+app.use("/", express.static(publicDir));
+
+// Set fallback application options
+app.use(fallback("index.html", { root: publicDir }));
+
+// Serve up application on specified port
+app.listen(process.env.PORT);
```

Comment:
From the node docs:
> If port is omitted or is 0, the operating system will assign an arbitrary unused port

we can pass a port with an environment variable or modify this script to set a default:

```
const PORT = process.env.PORT || 3000;
```

Reply:
It doesn't matter once in production what port it runs on.  Rather than hard-coding and/or setting in-app settings, I wonder if we just let the operating system pick.  this eliminates a configuration requirement currently, someone forgetting to set it.  I'm in favor letting OS pick.  What do you think.

Reply:
@chrisdholt what pros/cons do you see with allowing Node to choose the port in production?  It feels all upside for our production scenarios.

Reply:
Predictability and reproducibility are the main reasons to set a port. Once each package has its own `server.js` it will also allow us to use it for local testing (and will be really important when we move towards k8s, where it's proper to set the ports explicitly)